advanced/main.py

```diff
@@ -1,9 +1,12 @@
 import sys
 
-from sematic import CloudResolver
+from sematic import CloudRunner
 
 from advanced.pipeline import pipeline
 
 
 if __name__ == "__main__":
-    print(pipeline(sys.argv[1]).resolve(CloudResolver()))
+    future = pipeline(sys.argv[1]).set(
+        name="Advanced Docker Example Pipeline", tags=["examples", "docker", "advanced"]
```

Comment:
Our existing examples that use tags use `"example"` (singular) rather than `"examples"` (plural). Let's keep it consistent so looking up that tag gives all examples.

Reply:
Done.